mailbox/caching/src/main/java/org/apache/james/mailbox/caching/CachingMessageMapper.java

```diff
@@ -35,6 +35,11 @@ public CachingMessageMapper(MessageMapper underlying, MailboxMetadataCache cache
     }
 
     @Override
+    public Iterator<MessageUid> listAllMessageUids(Mailbox mailbox) throws MailboxException {
```

Comment:
sometimes comments don't appear at the right place
Ok, looks consistent with other methods 👍